browser/src/Performance.ts

```diff
@@ -1,28 +1,35 @@
 
+export interface IPerformance {
+    mark(markerName: string): void
+    startMeasure(measurementName: string): void
+    endMeasure(measurementName: string): void
+}
 /**
  * Thin wrapper around browser performance API
  */
-export function mark(markerName: string): void {
-    if (typeof window === "undefined") {
-        return
-    }
+export class Performance implements IPerformance {
```

Comment:
I really like the strategy overall of making the dependencies explicit - but I wonder for this class in particular if there is any benefit, as it is essentially wrapping the `console` object (which is itself a singleton).

Reply:
It is needed only for `Oni`-s mock, as it doesn't the the `performance` module (the one that is used on the non-mock version).